scripts/evergreen/teardown_kubernetes_environment.sh

```diff
@@ -5,6 +5,12 @@ set -Eeou pipefail
 source scripts/dev/set_env_context.sh
 
 if [ "${KUBE_ENVIRONMENT_NAME}" = "kind" ]; then
+    docker system prune -a -f
```

Comment:
for reviewer: this was in evergreen before but for minikube we rely on podman. We can also make this dependent on container runtime instead of kube env